contentrepo/urls.py

```diff
@@ -26,7 +26,7 @@
 
 urlpatterns = [
     path("django-admin/", admin.site.urls),
-    path("", include(wagtail_content_import_urls)),
```

Comment:
this is going to be removed in the next PR, we don't use wagtail content import anymore